docs/source/installing.rst

```diff
@@ -1,8 +1,6 @@
-.. _installing:
-
-***************
-Installing
-***************
+**************
+Installing XGI
```

Comment:
I feel like adding "XGI" makes it longer and the menu doesn't have so much space, so I think I wouldn't add it

Reply:
Sounds good. I'll change it back.

Reply:
Fixed.